src/core/rating_store.py

```
@@ -0,0 +1,203 @@
+                    record.price_input, record.price_output,
+                    limit_ratio, is_over_limit,
+                    record.input_mode, record.had_transcript,
+                    record.had_time_range, record.had_questions,
+                ),
+            )
+            return cursor.lastrowid
+
+    def update_quality_score(self, analysis_id: int, score: int) -> None:
+        """Setzt die manuelle Qualitätsbewertung (1-5)."""
+        if not 1 <= score <= 5:
+            raise ValueError(f"Score muss 1-5 sein, war: {score}")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)